operator/src/main/java/org/keycloak/operator/crds/v2alpha1/deployment/KeycloakSpec.java

```diff
@@ -105,6 +106,10 @@ public class KeycloakSpec {
     @JsonPropertyDescription("In this section you can configure Keycloak's reverse proxy setting")
     private ProxySpec proxySpec;
 
+    @JsonProperty("management")
```

Comment:
One last thing. :) Should it be like this?
```suggestion
    @JsonProperty("httpManagement")
```